db/dump.py

```diff
@@ -264,7 +387,13 @@ def import_db_dump(archive_path):
                         logging.info("Schema version verified.")
 
                 else:
-                    if file_name in table_names:
+                    if _is_partitioned_table_dump_file(file_name):
+                        table_name = member.name.split("/")[2]
+                        logging.info(" - Importing data from file %s into %s table..." % (file_name, table_name))
+                        cursor.copy_from(tar.extractfile(member), '"%s"' % table_name,
```

Comment:
We're streaming each file out of the archive file in order, which means we need to be absolutely sure that we apply multi-part files in the right order (lowlevel-1, lowlevel-2, lowlevel-3, etc). I think that the order when uncompressing will be the same as when we compress, but I can imagine that for some reason we uncompress/recompress a dump file and then find that the files are in some other order.
Can we have a basic assert that the suffix numbers apply in increasing order?